tests/e2e/kafka_cluster_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/banzaicloud/koperator/api/v1beta1"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+const (
+	kafkaKind        = "kafkaclusters.kafka.banzaicloud.io"
+	kafkaClusterName = "kafka"
+)
+
+// createKafkaClusterIfDoesNotExist creates a kafka cluster if
+// there isn't a preexisting one
+func createKafkaClusterIfDoesNotExist(kubectlOptions *k8s.KubectlOptions, path string) {
+	By("Checking existing kafkaClusters")
+	err := checkExistenceOfK8sResource(kubectlOptions, kafkaKind, kafkaClusterName)
+
+	if err == nil {
+		fmt.Printf("Kafka cluster %s already exists", kafkaClusterName)
```

Comment:
I think in the test we cannot write out things with fmt.Printf. This message will be hidden.

Reply:
It can:
```
STEP: Checking the existence of resource kafka @ 05/31/23 11:03:50.937
when Creating a simple kafka cluster Creating a Kafka cluster 2023-05-31T11:03:50+02:00 logger.go:66: Running command kubectl with args [--namespace kafka get kafkaclusters.kafka.banzaicloud.io kafka]
when Creating a simple kafka cluster Creating a Kafka cluster 2023-05-31T11:03:51+02:00 logger.go:66: NAME    CLUSTER STATE    CLUSTER ALERT COUNT   LAST SUCCESSFUL UPGRADE   UPGRADE ERROR COUNT   AGE
when Creating a simple kafka cluster Creating a Kafka cluster 2023-05-31T11:03:51+02:00 logger.go:66: kafka   ClusterRunning   0                                               0                     3m9s
Kafka cluster kafka already exists
```

Reply:
Though using By looks nicer, I'll change it later.